rust/rust.grammar/external/binaries-list

```diff
@@ -0,0 +1,23 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# The full antlr4 binary, as a build-time dependency (se use it to generate code from the grammar)
+844C603E04AB201B769849EE9D3CCE67BA7A1337 org.antlr:antlr4:4.11.1
```

Comment:
Just a nitpick, could you consider updating ANTLR to the latest version 4.12.0 (bug fix release).

Reply:
Good point, but the idea is to keep the compiler always in sync with the Antlr4 runtime version in  [ide/libs.antlr4](https://github.com/apache/netbeans/blob/master/ide/libs.antlr4.runtime/external/binaries-list) , which is still 4.11.1. We could bump that one too.